src/commonMain/kotlin/exchange.dydx.abacus/functional/vault/Vault.kt

```diff
@@ -114,7 +114,7 @@ object VaultCalculator {
             }
         }.filter { entry -> entry.date != null && entry.date >= dataCutoffMs }
 
-        val latestEntry = history.first()
```

Comment:
first() will throw exception when list is empty.

Reply:
oh very good call